Conference_Track_Management/TrackManager.cs

```diff
@@ -38,8 +38,9 @@ private Track AllocateProposalsToTrack(List<Proposal> proposals)
             var unAllocatedProposals = proposals;
             var allocatedProposals = new List<Proposal>();
             var allocatedProposalDuration = 0;
-            foreach (var proposal in unAllocatedProposals)
+            foreach (var proposal in unAllocatedProposals.ToList())
```

Comment:
Do we need ToList() here?

Reply:
Yes, as i am removing items from the unallocated proposals as they are allocated so i get an enumeration error if i am trying to enumerate on the same list